### PR TITLE
Fix doc for universes(foo) attributes

### LINKED ISSUE #10570: Universe attributes documentation is outdated
It seems that cc95e2f changed the name of universe-related attributes, but forgot to update the documentation. We are now in the unfortunate situation where only the deprecated syntax is documented.

Cc @SkySkimmer

### PR DESCRIPTION
**Pull request by SkySkimmer**
Fix #10570

### CODE REVIEW

File: doc/sphinx/language/gallina-specification-language.rst [hunk 1]
```diff
@@ -1503,19 +1503,18 @@ Each attribute has a name (an identifier) and may have a value.
...
-Currently,
-the following attributes names are recognized:
+Currently, the following attributes names are recognized:
 
-``monomorphic``, ``polymorphic``
-    Take no value, analogous to the ``Monomorphic`` and ``Polymorphic`` flags
-    (see :ref:`polymorphicuniverses`).
+``universes(monomorphic)``, ``universes(polymorphic)``
+    Take no value, equivalent to the ``Monomorphic`` and
+    ``Polymorphic`` flags (see :ref:`polymorphicuniverses`).
```
Comment (Zimmi48) [inline doc/sphinx/language/gallina-specification-language.rst]: I'd rather say that the `universes` attribute must take as value one of the optional sub-attributes `monomorphic`, `polymorphic`, `template`, `notemplate`.

**Review by Zimmi48 — COMMENTED** [review]
Thanks for that!

**Comment by Zimmi48** [comment]
@SkySkimmer Are you OK with my suggestion? Do you intend to implement it or would you like me to do it for you?

File: doc/sphinx/language/gallina-specification-language.rst (hunk 1 above)
Comment (SkySkimmer) [reply doc/sphinx/language/gallina-specification-language.rst]: Talking about template in this section seems wrong, it's pretty specific to Inductive/Record.

Comment (Zimmi48) [reply doc/sphinx/language/gallina-specification-language.rst]: IMO this section must present all the attributes that exist, and limit their scope by saying on which commands they apply. But also OK for merging as is if you want.

Comment (SkySkimmer) [reply doc/sphinx/language/gallina-specification-language.rst]: I changed the wording a bit to be closer to how I think of this stuff, have a look.

**Review by Zimmi48 — APPROVED** [review]
Looks better, thanks!

**Pull request merged**